src/main/java/com/smartbear/postman/PostmanImporter.java

```diff
@@ -461,4 +467,39 @@ private String convertTemplateProperties(String postmanUri) {
             return postmanUri.replaceAll("\\{\\{", "{").replaceAll("\\}\\}", "}");
         }
     }
+
+    private class PostmanImporterWorker implements Worker {
+        private String url;
+        private String postmanJson;
+
+        public PostmanImporterWorker(String url) {
+            this.url = url;
+        }
+
+        @Override
+        public Object construct(XProgressMonitor monitor) {
+            UrlClientLoader loader = new UrlClientLoader(url);
+            try {
+                loader.setUseWorker(false);
+                postmanJson = IOUtils.toString(loader.load(), StandardCharsets.UTF_8);
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
I think that we should show the error message using ``UISupport.showErrorMessage(e.getMessage());`` as well.

Reply:
@Alexander-Sviridov, ok